obspy/io/sac/tests/test_core.py

```diff
@@ -316,7 +316,7 @@ def test_undefinedB(self):
             self.assertEqual(tr2.stats.starttime.timestamp, 269596810.0)
```

Comment:
I am a bit too lazy to delve into this right now but I'll do it first thing week. Until then, here is the commit and the issue that led to that test: #107 and 450de3cdd1ac976bcfaec08b84308bae57ef8df3

Cheers!